packages/playground/src/index.ts

```diff
@@ -191,21 +191,21 @@ export const setupPlayground = (
       const lenses = !showFileCodeLens
         ? []
         : [
-          {
```

Comment:
Blah, this code wasn't formatted properly in the first place. I guess it's fine to do this here.